pkg/controller/controller.go

```diff
@@ -94,7 +96,12 @@ func (c *Controller) InstallKubeless(ctlImage string, ctlNamespace string) {
 	c.logger.Infof("Installing Kubeless controller into Kubernetes deployment...")
 	err := utils.DeployKubeless(c.Config.KubeCli, ctlImage, ctlNamespace)
 	if err != nil {
-		c.logger.Errorf("Kubeless controller installation failed: %v", err)
+
+		if strings.Contains(err.Error(), "already exists") {
```

Comment:
I think we can do better by checking k8s to see if the kubeless-controller pod already exists. Something like this but using Get instead of List

https://github.com/bitnami/kubeless/blob/master/pkg/utils/k8sutil.go#L605